llarp/iwp/session.cpp

```diff
@@ -298,15 +299,25 @@ namespace llarp
     util::StatusObject
     Session::ExtractStatus() const
     {
-      return {{"tx", m_CurrentTX},
-              {"rx", m_CurrentRX},
-              {"state", m_State},
+      const auto now = m_Parent->Now();
+
+      return {{"txRateCurrent", m_Stats.currentRateTX},
```

Comment:
can we keep `tx`/`rx` for backwards compat?

Reply:
`2279ebe` duplicates these. duplicating seems sloppy, but `tx` and `rx` are very ambiguous now that there are so many similar stats.